core/src/io/anuke/mindustry/core/Renderer.java

```diff
@@ -104,7 +104,9 @@ public void update(){
         Color.white.set(1f, 1f, 1f, 1f);
 
         camerascale = Mathf.lerpDelta(camerascale, targetscale, 0.1f);
-
+        
+        Bloom.setBloomIntesity(12f / camerascale);
```

Comment:
What if bloom is disabled, and/or bloom doesn't work on someone's machine?

Reply:
I think that this whole issue about bloom could be fixed by adjusting Gaussian blur size based on zoom, but that would need a different code for the Gaussian shader. Also, by disabling the original layer and displaying only the Gaussian layer, I've noticed that its much darker than the original, is this intentional or just an issue?

Reply:
That's either a side effect of the blurring taking into account nearby black clear pixels, or just how the threshold filter outputs the result.